orion/pkg/controllers/multiple_services.go

```diff
@@ -0,0 +1,56 @@
+package controllers
+
+import (
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/services"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func callServices() (domains.Domains, error) {
```

Comment:
Rename to `callGetUnpublishedServices()` or something signify we're specifically calling GetUnpublished for all services.